nuxeo-features/nuxeo-platform-mail/nuxeo-platform-mail-core/src/test/resources/OSGI-INF/nxmail-core-contrib.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<component
+  name="org.nuxeo.ecm.platform.mail.core.types.contrib.test">
+
+  <require>org.nuxeo.ecm.platform.mail.core.types.contrib</require>
+
+  <extension target="org.nuxeo.ecm.core.schema.TypeService"
+    point="doctype">
+
+    <doctype name="CustomMailMessage" extends="MailMessage" />
+  </extension>
+</component>
```

Comment:
Could you add a new line at the end of the file?

Reply:
Done